### DISCUSSION TITLE
Using reactivated to power dynamic forms

### DISCUSSION THREAD

**Discussion by zusoomro** [silviogutierrez/reactivated#227] [Q&A] [answered]
Hey! I’m excited to use reactivated, but I’m not sure that I understand if theres a way to render dynamic forms. Could someone point me to an example of a dynamic form using the <Form/> tag? (By dynamic form, I mean a form which has a number of fields that can be changed by the user.)

**Comment by silviogutierrez** [comment, marked as answer]
Hi there,

Absolutely you can have dynamic forms, that's one of the key features.

Check out the docs here, specifically the bottom where country is checked: https://www.reactivated.io/documentation/api/#useform

As you can see, this is mostly *presentational*, but it'll achieve what you want.

Hope that helps.
